source/Core/SystemRequirements.php

```diff
@@ -284,14 +284,13 @@ public function checkBug53632()
             return 2;
         }
 
-        $iState = 1;
         if (version_compare($this->getPhpVersion(), "5.3", ">=")) {
             if (version_compare($this->getPhpVersion(), "5.3.5", ">=") && version_compare($this->getPhpVersion(), "5.3.7", "!=")) {
-                $iState = 2;
+                return 2;
```

Comment:
We should change either this or [this](https://github.com/OXID-eSales/oxideshop_ce/pull/414/files#diff-cc05b37d89e5a6f6e8c966e4cb77b670R284)